app/models/episode.rb

```diff
@@ -207,6 +218,48 @@ def media_files
     end
   end
 
+  def media_files=(files)
```

Comment:
yes, just via the API - I thought about creating an api handler (like the ones for crier and cms) since this is similar, but ended up deciding the API updates should stay here as they are more central or essential to the app (this is the app's api in its own format, not data coming from another system in that systems format).